packages/rum-core/src/domain/foregroundContexts.spec.ts

```diff
@@ -0,0 +1,280 @@
+import { RelativeTime, relativeNow, ServerDuration, toServerDuration, Duration } from '@datadog/browser-core'
+import { setup, TestSetupBuilder } from '../../test/specHelper'
+import { createNewEvent } from '../../../core/test/specHelper'
+import { InForegroundPeriod } from '../rawRumEvent.types'
+import { startForegroundContexts, ForegroundContexts } from './foregroundContexts'
+
+const FIVE_SECONDS_MS = 5_000 as Duration
```

Comment:
using ms directly could spare you a bunch of `_000`, reducing a bit of noise

Reply:
How? Do you have an example somewhere?

Reply:
my point is that your test don't seem to need to manipulate values in seconds.
So you could directly have:
```ts
const FOCUS_PERIOD_LENGTH = 5
clock.tick(1)
foregroundContext.getInForeground(2 as RelativeTime)
```